src/test/ui/unsized-locals/unsized-locals-using-unsized-fn-params.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(unsized_fn_params)]
+
+// Ensure that even with unsized_fn_params, unsized locals are not accepted.
```

Comment:
What is the reason to have this in a separate test from unsized-local-pat.rs? The purpose seems to be the same -- make sure the feature gate does not accept too many things.

Reply:
:+1:, I've merged both tests.